model/src/main/java/org/mskcc/cmo/metadb/model/MetadbRequest.java

```diff
@@ -385,6 +386,16 @@ public void updateRequestMetadataByMetadata(RequestMetadata requestMetadata)
         addRequestMetadata(requestMetadata);
     }
 
+    public String resolveIgoRequestId(Map<String, Object> metadataMap) {
```

Comment:
just now noticed that these aren't in the RequestDataFactory class - we want to try and minimize any of this kind of logic/data extraction outside of the models

Reply:
keeping this for now